### PR TITLE
[TK-8]: create terms and condition screen.

### PR DESCRIPTION
**Pull request by waqarunnisa**
- Add terms and condition screen.
- Add TermsAndConditionvVewController.
- Add TermsAndConditionStoryboard.
- Add corner radius of next button.
- set constraints of the screen.

### CODE REVIEW

File: TermsAndConditionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        readMoreView.layer.cornerRadius = 15
+         textView.isScrollEnabled = true
+        textView.text = "Lorem ipsum dolor sit er elit lamet, consectetaur cillium adipisicing pecu, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Nam liber te conscient to factor tum poen legum odioque civiuda Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Nam liber te conscient to factor tum poen legum odioque civiuda."
+    }
+
+    @IBAction func readMoreButton(_ sender: Any) {
+        
+    }
+    
+
```
Comment (MuhammadHuzaifa2) [inline TermsAndConditionViewController.swift:29]: remove extra spacing

Comment (waqarunnisa) [reply TermsAndConditionViewController.swift]: Done

File: TermsAndConditionViewController.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+//  Created by Mehar on 9/22/22.
+//
+
+import UIKit
+
+class TermsAndConditionViewController: UIViewController {
+    
+    @IBOutlet weak var textView: UITextView!
+    @IBOutlet weak var NextButtonOutlet: UIButton!
+    @IBOutlet weak var readMoreView: UIView!
```
Comment (ammadahmed22) [inline TermsAndConditionViewController.swift:14]: Why is the readMore a label and not a button? Also, the arrow is pointing the wrong way.

File: Vape-X/Info.plist [hunk 3]
```diff
@@ -16,7 +16,7 @@
 					<key>UISceneDelegateClassName</key>
 					<string>$(PRODUCT_MODULE_NAME).SceneDelegate</string>
 					<key>UISceneStoryboardFile</key>
-					<string>Main</string>
```
Comment (ammadahmed22) [inline Vape-X/Info.plist:19]: This need to stay as Main.

File: TermsAndConditionStoryboard.storyboard [hunk 4]
```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="21225" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="PVx-mf-bBp">
```
Comment (ammadahmed22) [inline TermsAndConditionStoryboard.storyboard:2]: Your TermsAndConditionStoryboard shouldn't be the Initial View Controller.

**Pull request closed without merging**